_component_design/chips.md

```diff
@@ -17,27 +17,99 @@ helix-ui-javascript: false
 
 <section class="static-section" markdown="1">
 
```

Comment:
Just a note - Design Guidelines above the contents list (When to use, Best practices, and ) does not use sentence-style capitalization. Should it be Design guidelines?

Reply:
you are correct, that is part of the page template - I'll mention it to the team. Thanks!